carla/recourse_methods/autoencoder/models.py

```diff
@@ -0,0 +1,115 @@
+import os
+
+from keras import backend as K
+from keras.layers import Dense, Input
+from keras.models import Model, Sequential
+
+
+class Autoencoder:
+    def __init__(
+        self, dim_input, dim_hidden_layer1, dim_hidden_layer2, dim_latent, data_name
+    ):
+        """
+        Defines the structure of the autoencoder
+        :param dim_input: int > 0; number of neurons for this layer (for Adult: 104)
+        :param dim_hidden_layer_1: int > 0, number of neurons for this layer (for Adult: 30)
+        :param dim_hidden_layer_2: int > 0, number of neurons for this layer (for Adult: 15)
```

Comment:
Rather then putting it in comments could we put the configs for different datasets in dictionary's?